synapse/api/constants.py

```diff
@@ -120,6 +120,7 @@ class EventTypes:
     SpaceParent = "m.space.parent"
 
     MSC2716_INSERTION = "org.matrix.msc2716.insertion"
+    MSC2716_CHUNK = "org.matrix.msc2716.chunk"
```

Comment:
Feel free to suggest better names!

Maybe we want to rename `insertion` to `chunk_end` so we have `chunk_start` and `chunk_end` events 🤷‍♀️ (which one is the start vs end 😜)